synphot/spectrum.py

```diff
@@ -778,6 +778,67 @@ def plot(self, wavelengths=None, **kwargs):  # pragma: no cover
         w, y = self._get_arrays(wavelengths)
         self._do_plot(w, y, **kwargs)
 
+    @classmethod
+    def from_Spectrum1D(cls, spec, keep_neg=False):
+        """Create a spectrum from `specutils.Spectrum1D` object.
+
+        Parameters
+        ----------
+        spec : `specutils.Spectrum1D`
+
+        keep_neg : bool
+            See `~synphot.models.Empirical1D`.
+
+        Returns
+        -------
+        sp : `BaseSourceSpectrum`
+            Empirical spectrum.
+
+        """
+        # Spectrum1D is designed to be immutable, so no need to make
+        # copies of spectral_axis nor flux.
+        return cls(Empirical1D, points=spec.spectral_axis,
```

Comment:
This doesn't seem to do anything with `mask`. Am I right that `Empirical1D` has no concept of a mask?  If so, it probably makes sense to apply the mask to the spectral points if they exist.  Something like:
```suggestion
        if spec.mask is not None:
            msk = spec.mask == False
            lookup_table = spec.flux[msk]
            points = spec.spectral_axis[msk]
        else:
            lookup_table = spec.flux
            points = spec.spectral_axis
        return cls(Empirical1D, points=points, lookup_table=lookup_table,
```

?

Reply:
Oh, I am not aware you have masking. I took the API off Getting Started and did not dig too deep into `specutils`. 😬  I'll add this in. Thanks!

Reply:
Wait a minute... Why isn't the mask automatically applied by `specutils` when you access the `flux` or `spectral_axis` attributes?

Reply:
Anyway, support for mask is added (for now).